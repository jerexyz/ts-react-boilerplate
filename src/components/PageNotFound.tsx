import * as React from 'react';

const PageNotFound: React.StatelessComponent<undefined> = () =>
    <div className="page-not-found">404 - page not found</div>;

export default PageNotFound;
