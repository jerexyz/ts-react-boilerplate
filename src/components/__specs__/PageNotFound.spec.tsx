import * as React from 'react';
import { shallow } from 'enzyme';
import PageNotFound from '../PageNotFound';

describe('PageNotFound', () =>
    it('should render correctly', () => expect(shallow(<PageNotFound />)).toMatchSnapshot()));
