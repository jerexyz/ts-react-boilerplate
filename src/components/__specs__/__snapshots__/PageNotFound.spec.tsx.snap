// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageNotFound should render correctly 1`] = `
<div
  className="page-not-found"
>
  404 - page not found
</div>
`;
